### PR TITLE
Different fix for Issue #9 that doesn't clobber tests

### PR DESCRIPTION
**Pull request by voutilad**
Had to rework the test logic to match the new approach to returns from the `convert_images.main` function.

The `test_convert_images_tool_accepts_args` test now uses regex to parse out the results from stdout, which I think is fine given that test is supposed to be testing the tooling from the user's point of view. Checks for return code being zero as both executions should be successful.

### CODE REVIEW

File: seal_rookery/convert_images.py [hunk 1]
```diff
@@ -124,13 +124,14 @@ def main(argv=None):
...
-    )
-
-    save_new_json()
+    try:
+        changed, skipped = convert_images(
+            verbose=bool(args.v), forced=bool(args.f)
+        )
+        save_new_json()
+    except Exception:
+        return 1
```
Comment (mlissner) [inline seal_rookery/convert_images.py]: Should we print the exception here?

Comment (voutilad) [inline seal_rookery/convert_images.py]: Might be helpful. I'm honestly not sure what Exceptions could occur, but I wanted to make sure a non-zero code is returned.

Comment (mlissner) [inline seal_rookery/convert_images.py]: My worry is that a non-zero exit code is really easy to miss, so if no exception and you miss it, you'll think things worked properly.

Comment (voutilad) [inline seal_rookery/convert_images.py]: Hmm let me try something. I was going to add a test for bad argparse input.

Comment (voutilad) [inline seal_rookery/convert_images.py]: See latest commit: `cc46426`

**Pull request merged**